tbb-toolfile.spec

```diff
@@ -13,8 +13,8 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/tbb.xml
   <lib name="tbb"/>
   <client>
     <environment name="TBB_BASE" default="@TOOL_ROOT@"/>
-    <environment name="LIBDIR" default="$TBB_BASE/lib"/>
-    <environment name="INCLUDE" default="$TBB_BASE/include"/>
+    <environment name="LIBDIR"   default="$TBB_BASE/lib"/>
```

Comment:
You might need to change this to lib64.

Reply:
Nevermind. I see you set CMAKE_INSTALL_LIBDIR=lib

Reply:
we are having this policy of deliberately keeping the lib instead of lib64